src/widget/weffectpushbutton.cpp

```diff
@@ -64,7 +64,7 @@ void WEffectPushButton::mousePressEvent(QMouseEvent* e) {
     // The push handler may have set the left value. Check the corresponding
     // QAction.
     double leftValue = getControlParameterLeft();
-    foreach (QAction* action, m_pButtonMenu->actions()) {
+    for (const auto& action : m_pButtonMenu->actions()) {
```

Comment:
Derp. I did this manually (since it was a foreach). Replaced with QAction* for readability.